README.rst

```diff
@@ -37,6 +37,72 @@ Alternatively to get more up-to-date features:
 
 check the changelog_ for information about features included on `master` but not yet released.
 
+Dev Env Setup
+-------------
+
+Installation:
+    Python https://www.python.org/
+
+    Pyenv https://github.com/pyenv/pyenv#installation
+
+Python Version:
+    Current version accepted for development is v3.5.6
+
+Install Python versions:
+    Use pyenv to install multiple Python versions, specify global, local or
+    temporary Python version, and to switch between different versions.
+
+    To install different versions of Python, and to list installed pyenv
+    versions:
+
+    .. code-block:: bash
+
+        pyenv install <python_version>
+        pyenv versions
+
+    To activate pyenv:
```

Comment:
While we use pyenv internally I'm not sure how I feel about pushing it along with APPY. Other folks may very well be using pipenv, conda (very likely among APPY users), or some other alternative.

Reply:
hmm... I guess I did initially install conda before I was told to use pyenv. lol. Will bring this up so I can get other opinions. Otherwise, we can mark python version as requirement, and have pyenv, etc, as optional/suggested setup to leave it up to the developer?